beacon-chain/rpc/validator/aggregation.go

```diff
@@ -0,0 +1,82 @@
+package validator
+
```

Comment:
Would you like to name the file `aggregator.go` instead of `aggregation.go`?  It aligns better with `proposer.go` and `attester.go`

Reply:
sounds good